phalcon/http/request.zep

```diff
@@ -320,6 +324,15 @@ class Request implements RequestInterface, InjectionAwareInterface
 	}
 
 	/**
+	 * Set request raw body for unit test
+	 */
+	public function setRawBody(string rawBody) -> <Request>
+	{
+	    let this->_rawBody = rawBody;
+	    return this;
+	}
```

Comment:
Thanks for the solution, I removed setRawBody method and make unit test as your way